c10/util/Array.h

```diff
@@ -0,0 +1,16 @@
+#include <array>
+#include <utility>
+
+namespace c10 {
+
+// This helper function creates a constexpr std::array
+// From a compile time list of values, without requiring you to explicitly
+// write out the length.
+//
+// See also https://stackoverflow.com/a/26351760/23845
+template <typename V, typename... T>
```

Comment:
I have some TODO comment lying around somewhere but when we switch to c++20 there will be a stdlib version